codegens/js-xhr/test/unit/convert.test.js

```diff
@@ -242,6 +242,47 @@ describe('js-xhr convert function', function () {
       });
     });
   });
+
+  describe('Request with no body', function () {
+    var req = {
+        'method': 'GET',
+        'url': {
+          'raw': 'https://postman-echo.com/get',
```

Comment:
Although it doesn't change the output, the path should be https://postman-echo.com/post @umeshp7

Reply:
A GET call to https://postman-echo.com/post ?

Reply:
Ah, missed the path. 
I changed the path property to 'get'. Since I am making a GET call to `https://postman-echo.com/get`.